src/app/tests/crud/test_crud_welding_point.py

```diff
@@ -0,0 +1,90 @@
+import pytest
+import sqlalchemy
+from sqlalchemy.exc import IntegrityError
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from app.models.project import Project
+from app.models.welding_point import WeldingPoint
+from app.crud.crud_welding_point import welding_point
+from app.crud.crud_project import project
+from app.schemas.welding_point import WeldingPointUpdate
+from app.tests.utils.models import create_welding_point, create_project, get_welding_point_create
+
+pytestmark = pytest.mark.asyncio
+
+
+async def test_crud_welding_point_create_fail(database: AsyncSession):
+    project_obj = Project(id=1, name="Test")
+    try:
+        await create_welding_point(db=database, project_obj=project_obj)
+        assert False
+    except IntegrityError:
+        assert True
+
+
+async def test_crud_welding_point_create(database: AsyncSession):
+    project_obj = await create_project(db=database)
+
+    welding_point_obj = await create_welding_point(db=database, project_obj=project_obj)
+    assert welding_point_obj is not None
+    assert isinstance(welding_point_obj, WeldingPoint)
```

Comment:
Please also check the that the object has arrived in the database. (And not just that I return the request body and still get a pass :wink:)

Reply:
Addressed